docs/advanced-solidus/returns.mdx

```diff
@@ -0,0 +1,117 @@
+
+An RMA must reference one or more of the products a customer purchased. For
+instance, a user bought five cups, but one of them arrived damaged. Depending
+on the store policy, they might ask for a reimbursement or an exchange for
+another cup. At a later stage, the request will be approved, edited or denied.
+
+In Solidus, each item is represented through a
+[`Spree::ReturnItem`][return-item] record. Notice that a `Spree::ReturnItem`
+instance is associated with a [`Spree::InventoryUnit`][inventory-unit]. That's
+the most concrete level of product identification in orders. I.e., orders can
+have several line items for different variants, and each line item may contain
+one or more. The inventory unit maps one to one to a physical (or digital, for
+what it's worth) object, such as a broken cup.
+
+A `Spree::ReturnItem` contains information about the requested reimbursement
+amount and how it should be processed (e.g., refund or store credit) or another
+unit with which the store should replace it.
+
+Finally, two status fields govern the lifecycle of an item for return:
+
+- The **acceptance status** controls the eligibility of the return per-se. Initially
+_pending_, it should end up _accepted_ in case of going forward.
+
+- The **reception status** marks whether the store has already received the
+returned package. In a standard scenario, it should transition from _awaiting_
+to _received_.
+
+## Customer return
+
+Finally, after the return authorization has moved forward, it'll materialize in
+a customer return. At that point, a store will already have the returned items
+back. It might be that the customer support department rejected some of the
+requests or accepted all of them.
+
+In Solidus, that's represented by a [`Spree::CustomerReturn`][customer-return]
+record. Similar that RMAs, it's a simple model. It only contains an
+identification number and a stock location that is as a final override of the
+RMA's proposal. Customer returns are flexible and not tied to a single return
+authorization. The important association here is that a `Spree::ReturnItem`
+belongs to a `Spree::CustomerReturn`.
+
+## Reimbursements
+
+The final step is fulfilling the store obligations and performing the agreed
+reimbursement or exchange. The store will need another shipment with the new
+items in the latter case.
+
+[`Spree::Reimbursement`][reimbursement] takes care of wrapping everything up.
+It's associated with a `Spree::CustomerReturn`, but the system is flexible
+enough so that more than one can be created until all returned items have been
+processed.
+
+Typically, administrators can edit the returned item details, like the amount
+to refund, the article to use as a substitute, or if it should be marked as
+resellable. The system will also take care of creating a new shipment if
+needed. In the end, a new reimbursement record will be created with a state of
+either reimbursed or errored.
+
+## How-to guides
+
+- [How to skip RMAs][how-to-skip-rmas]
```

Comment:
This is good enough for now but, if we want to scale it, I think we might use some sort of tagging system on the how-tos and dynamically display all the how-tos with certain tag(s). It will take some extra technical work, so I will create a ticket to tackle at a later stage.